generators/container/templates/container.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './<%= _.kebabCase(componentName) %>.css';
+
+function mapStateToProps(state) {
+  return {
+
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+
+  };
+}
+
+@connect(mapStateToProps, mapDispatchToProps)
```

Comment:
Yes, I suggested remove this with the stage-0. Two features that I will miss:

- decorators
- auto bind with `::`